website/vercel.json

```diff
@@ -408,7 +408,8 @@
     },
     {
       "source": "/datasource-reference/querying-mongodb/whitelist-mongodb-cloud",
-      "destination": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud"
+      "destination": "/connect-data/reference/querying-mongodb",
```

Comment:
The redirect rules should be added as new. These redirect rules are needed as is.